Let the pull requests begin (none of these are finished, I'm going to…

kduncan-personal opened the pull request:
… wait for comments).

Mostly adding new content pictures and stuff should be the real comment from me.

Commit:
Let the pull requests begin (none of these are finished, I'm going to wait for comments).

edanaher commented on Toby.html:
```diff
@@ -0,0 +1,36 @@
+
+		</div-->
+	</div>
+</div>
```
Tsk, tsk.  No newline at end of file?

edanaher commented on boston.html:
```diff
@@ -16,8 +16,78 @@
+						<p>You may just be in Boston for a couple day's, but we've lived here for a couple years (and Kelly grew up here).  So here are some of our suggestions for things to do if you have some free time while you're in town.  Unfortunately, some are in pseudo-latin.</p>
+						<h2>North End</h2>
+						<h4>Museum of Science</h4>
+						<p> Technically it's not in the North End, but that is what it is closest too.  This is probably Boston's best museum, and if you hit on this weekend, I'd hit this one</p>
```
"if you hit on this weekend, I'd hit this one" - I assume that's supposed to be "hit one", and you're being inconsistent with you/I.

(the first is serious, the second less so)

kduncan-personal replied:
or I just can't spell.

edanaher committed:
Minor edits, because it's nicer than commenting on every little thing.

edanaher commented:
You mostly use Kelly and Evan, but occasionally use "I".  Was that intentional?

Commit:
Lots of changes, mostly for sidebars reasons or aesthetic reasons, or because kelly doesnt like grammah.

Real reason is mostly switching sidebars so there is one for just the boston page.

Also adding Toby's schedule page for when we want to work on that.

H2 changed so text size overwrites in CSS.

Commit:
Merge branch 'kelly' of https://github.com/edanaher/ceremony-website into kelly

Conflicts:
	boston.html

edanaher committed:
Factor out Boston page into lots of tiny place files.

Commit:
Lots of content adds.

Commit:
Deleting 80_furtherwest, and as it is now 90_furtherwest

Commit:
More content updates

Commit:
A few more content changes

Commit:
Content Changes, and renames

Commit:
Embeds and other stuff

edanaher committed:
Add 00_place content so the maps show up.

Commit:
Changes to boston page to create new england page.  Lots of content updates

Commit:
Merge branch 'kelly' of https://github.com/edanaher/ceremony-website into kelly

Commit:
Merge Congestion adds

edanaher committed:
Fix Bradley link, some typos/style cleanups.

Commit:
More Content, also switching from places to boston and new england places

edanaher committed:
Various structural and attributive updates.

edanaher committed:
Fix some text, remove gratuitious vertical padding on the index page.

edanaher committed:
Make h3/h4 headers.  Makes lists much more legible.

edanaher committed:
Fixes for mobile css.

edanaher committed:
add images

Commit:
Deleting old files, updated for hotel blocks

Commit:
Adding link for Holiday Inn Somerville

edanaher committed:
Cleanups for styling.

edanaher committed:
Move text and us back to center

edanaher committed:
Don't use fontawesome.

edanaher committed:
Explicity use Roboto from Google.

edanaher committed:
More fixups!

edanaher committed:
Style cleanups for boston and schedule.

edanaher committed:
Fix README: s/nix-env/nix-shell/

edanaher committed:
Minor edits on the front page.

edanaher committed:
s/talk/contact, remove quotes around "I still need treats"

Commit:
Updating Schedule Page to most recent information.

Commit:
Registry stuff.

edanaher committed:
Tweak registry page and add link to Evan's.

edanaher committed:
Last minute timing updates.
